tempesta_fw/ss_skb.c

```diff
@@ -860,10 +871,26 @@ int
 ss_skb_get_room(struct sk_buff *skb_head, struct sk_buff *skb, char *pspt,
 		unsigned int len, TfwStr *it)
 {
-	int r = skb_fragment(skb_head, skb, pspt, len, it);
+	int _;
+
+	return ss_skb_get_room_w_frag(skb_head, skb, pspt, len, it, &_);
```

Comment:
Please move `ss_skb_get_room()` using `ss_skb_get_room_w_frag()` after the `ss_skb_get_room_w_frag()` definition.